tutorials/audio/audio_buses.rst

```diff
@@ -167,7 +165,7 @@ This effect allows for modulating pitch independently of tempo. All frequencies
 Reverb
 ~~~~~~
 
-Reverb simulates rooms of different sizes. It has adjustable parameters that can be tweaked to obtain the sound of a specific room. Reverb is commonly outputted from Areas (see :ref:`doc_audio-buses` tutorial, look for the section on Areas), or
+Reverb simulates rooms of different sizes. It has adjustable parameters that can be tweaked to obtain the sound of a specific room. Reverb is commonly outputted from :ref:`Areas <class_Area>` (see :ref:`doc_audio-buses` tutorial, look for the "Reverb buses" section), or
```

Comment:
Welp, this should point to `doc_audio-streams` as well, I was trying to link directly to the reverb section there but couldn't find out how and then forgot to change the actual link :/
@mhilbrunner can you change it please or should I make a second PR?

Reply:
@merumelu A second PR wiuld be great, but if not I'll fix it myself tomorrow, missed that too. Thanks :)

Reply:
@mhilbrunner #1887